src/iohumdrum.cpp

```diff
@@ -18937,8 +18937,8 @@ void HumdrumInput::convertNote(Note *note, hum::HTp token, int staffadj, int sta
         if (showInAccidGes) {
             if (loaccid.empty()) {
                 switch (accidCount) {
-                    // case +3: note->SetAccidGes(ACCIDENTAL_GESTURAL_ts); break;
-                    // case -3: note->SetAccidGes(ACCIDENTAL_GESTURAL_tf); break;
+                    case +3: note->SetAccidGes(ACCIDENTAL_GESTURAL_ts); break;
+                    case -3: note->SetAccidGes(ACCIDENTAL_GESTURAL_tf); break;
```

Comment:
This fails. It should be `accid` and not `note`

Reply:
Obviously, yes. I leave it to Craig.